onnxruntime/core/optimizer/identical_children_consolidation.cc

```diff
@@ -61,7 +61,7 @@ std::vector<std::vector<NodeIndex>> IdenticalChildrenConsolidation::DivideIdenti
 }
 
 string_view IdenticalChildrenConsolidation::IdentityBuilder(const Graph& graph, const Node& node) const {
-  std::string identity;
+  static std::string identity;
```

Comment:
Try to not use a thread-local variable. If you do this, this function will not be thread-safe.